index.html

```diff
@@ -0,0 +1,712 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mitra UI</title>
+    <link rel="stylesheet" href="components.css">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/v4-shims.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Varela+Round&display=swap">
+</head>
+
+<body>
+    <header class="page-main-menu">
+        <img src="./images/hands-peace-bnw.png" alt="logo" class="hero-image">
+        <div class="hero-name">Mitra UI</div>
```

Comment:
The logo also has a hover effect which is a little bit wierd or you may add a link to the index page on clicking the logo.
If I keep my mouse pointer on the logo itself its hovering infinitely,  you can look into this matter.